pkg/usernames/usernames.go

```diff
@@ -0,0 +1,19 @@
+package usernames
+
+// Response represents the result of a search request done by using the MUR name or an email address.
+// The result will specify if a user exits in the system or not , or if there are multiple matches for the given search query
+type Response struct {
+	// For is the search string that was specified in the search request url. Example: /usernames/<search string>
+	For string `json:"for"`
```

Comment:
I used the original names for the design doc since maybe there is already an agreement with RHTAP on those.
But personally I would suggest to use `Query` or `Search` here.

Reply:
The example indicates that <search string> is a path parameter so maybe it should be named according to that? eg. `UsernameParam`.

Reply:
That's assuming the search string is even needed in the response. It's kind of weird because the request contains the same information so not sure why it needs to be duplicated in the response but I'm guessing there was a reason that was discussed and agreed upon with RHTAP folks?

Reply:
> That's assuming the search string is even needed in the response.

I was thinking about the same thing. Maybe we can avoid it, since it should be already known to the client/UI that is sending the request.

Reply:
Keeping `for` in the struct would make sense if it was a Kube API. But for a custom REST API as this I don't see any value in it.

Reply:
Agree with that - it doesn't make much sense to keep it there. Didn't realize it when I put it into the doc. Thanks